handlers/renewmsg.go

```diff
@@ -0,0 +1,25 @@
+package handlers
+
+import "time"
+
+// RenewMsg is used to refresh the current view of the cluster in discovery service
+type RenewMsg struct {
+	// ClusterID is the cluster's unique identifier
+	ClusterID string `json:"cluster-id"`
```

Comment:
use camel case to be consistent with the json casing used throughout castle. For example, it would translate to clusterId, requestTime, and leaderName.